test/verify/check-kubernetes

```diff
@@ -20,8 +20,8 @@
 
 import unittest
 
-import kubelib
 import parent
+import kubelib
```

Comment:
In order to avoid reintroducing this with the next automatic sorting, can you please put the `import parent` into its own paragraph,  i. e. leave a newline after it?

Reply:
done